pkgs/development/libraries/libpointmatcher/default.nix

```diff
@@ -0,0 +1,38 @@
+{stdenv, fetchgit, cmake, eigen, boost, libnabo}:
+
+let inherit (stdenv.lib) getVersion versionAtLeast; in
+
+assert versionAtLeast (getVersion eigen) "3";
+assert versionAtLeast (getVersion boost) "1,48";
+assert versionAtLeast (getVersion eigen) "1.0.6";
+
+stdenv.mkDerivation rec {
+  version = "1.2.3";
+  name = "libpointmatcher-${version}";
+
+  src = fetchgit {
+    url = git://github.com/ethz-asl/libpointmatcher.git;
+    rev = "4e03f62bda50b9d33992f2e3f2286dc57ec2fa2b";
+    sha256 = "1bg6x6gxnl3748454wxbkn0m3jmnwn7gccyllbxl05ph8zc36icq";
+  };
+
+  buildInputs = [cmake eigen boost libnabo];
+
+  enableParallelBuilding = true;
+
+  inherit eigen;
+  cmakeFlags = "
+    -DEIGEN_INCLUDE_DIR=${eigen}/include/eigen3
+  ";
+
+  # doCheck = true;
+  # checkTarget = "test";
+  # TODO: https://github.com/ethz-asl/libpointmatcher#testing
```

Comment:
@groxxda I'm getting `cd build: No such File ...` after the build phase.

I had assumed it the check phase working directory would start in the source directory?

I tried with `nix-build -K -A libpointmatcher` and inspected the `/tmp/nix-build-...` location. `build/utest/utest` is definitely there. Do I need to use some variable like `cd ${foo}/build` ?